src/fu-engine.c

```diff
@@ -2057,8 +2057,25 @@ fu_engine_get_result_from_app (FuEngine *self, AsApp *app, GError **error)
 
 	/* verify trust */
 	release = as_app_get_release_default (app);
-	if (!fu_keyring_get_release_trust_flags (release, &trust_flags, error))
-		return NULL;
+	g_autoptr(GError) error_local = NULL;
+        if (!fu_keyring_get_release_trust_flags (release,
+                                                 &trust_flags,
+                                                 &error_local)) {
+                AsChecksum *csum = as_release_get_checksum_by_target (release,
+                                                    AS_CHECKSUM_TARGET_CONTENT);
+                if (g_error_matches (error_local,
+                                     FWUPD_ERROR,
+                                     FWUPD_ERROR_NOT_SUPPORTED) \
+                                     && csum != NULL) {
```

Comment:
why checking for `csum != NULL`?

Reply:
As a safety net in case as_release_get_checksum_by_target returns NULL. Would "(null)" be displayed if the csum check is removed?